src/richie/apps/search/filter_definitions/base.py

```diff
@@ -281,25 +280,34 @@ def get_faceted_definitions(self, facets, data, *args, **kwargs):
         # Check if there are more available values than we're about to return on the API
         has_more_values = len(facet_counts) > len(facet_counts_dict)
 
+        # Add human names to keys and counts before sorting as some of our sortings
+        # use alphabetical ordering.
+        values = [
```

Comment:
I found it hard cognitively that static values are a couple machine name/human name and that values are a dictionary with these 3 keys... 
I propose that `static_values` is renamed to `human_names` in this method.